### ISSUE TITLE
Terraform Crashes

### ISSUE THREAD

**Issue by dbadola** [terraform-providers/terraform-provider-fortios#5]
Trying to create multiple service port

```
module "config_firewall_service" {
  source = "./modules/service_port"

  service_port = [
    {
      name           = "tcp8110"
      protocol       = "TCP"
      tcp_port_range = "8110"
      udp_port_range = ""
      icmptype       = ""
      icmpcode       = ""
      comment        = "Proxy"
    },
    {
      name           = "udp8110"
      protocol       = "UDP"
      tcp_port_range = ""
      udp_port_range = "8110"
      icmptype       = ""
      icmpcode       = ""
      comment        = "Proxy"
    },
    {
      name           = "icmp3"
      protocol       = "ICMP"
      tcp_port_range = ""
      udp_port_range = ""
      icmptype       = "3"
      icmpcode       = "3"
      comment        = "Proxy"
    },
  ]
}
```

With the module 
```
resource "fortios_firewall_object_service" "port" {
  count         = "${length(var.service_port)}"
  name          = "${lookup(var.service_port[count.index],"name")}"
  category      = "General"
  protocol      = "${lookup(var.service_port[count.index],"protocol")}"
  tcp_portrange = "${lookup(var.service_port[count.index],"tcp_port_range")}"
  udp_portrange = "${lookup(var.service_port[count.index],"udp_port_range")}"
  icmptype      = "${lookup(var.service_port[count.index],"icmptype")}"
  icmpcode      = "${lookup(var.service_port[count.index],"icmpcode")}"

  comment = "${lookup(var.service_port[count.index],"comment")}"
}

Getting Terraform Crash again and again

```
!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Attached the crash.log file
[crash.log](https://github.com/terraform-providers/terraform-provider-fortios/files/3243231/crash.log)

Terraform --version is - 0.11

**Comment by frankshen01** [comment]
hi, @dbadola, thanks for reporting this issue and for detailing your workaround. 

When the protocol is set to "ICMP" for "fortios_firewall_object_service", the protocol_number need to set too now. Can you please try : 

    {
      name           = "icmp3"
      protocol       = "ICMP"
      tcp_port_range = ""
      udp_port_range = ""
      icmptype       = "3"
      icmpcode       = "3"
      **protocol_number = "1"**
      comment        = "Proxy"
    }

**Comment by frankshen01** [comment]
This has been released in version 1.1.0 of the Terraform FortiOS provider.
